extensions/exchanges/bittrex/exchange.js

```diff
@@ -1,9 +1,8 @@
-var bittrex_authed = require('node.bittrex.api'),
-    bittrex_public = require('node.bittrex.api'),
-    path = require('path'),
-    moment = require('moment'),
-    n = require('numbro'),
-    colors = require('colors')
+var bittrex_authed = require('node-bittrex-api'),
```

Comment:
Should `historyScan` still be `forward`?
https://github.com/DeviaVir/zenbot/pull/1215/files#diff-b3b56e634e6ef011a9923a3c54c40108L51

Reply:
Unsure.   The market history  on bittrex is very limited.   It will pull only the last couple of trades.  there is a workaround that someone came up with using the v2 api that seems to get around this limitation,  I'll look at implementing it.  hate having to leave a marking noop trading for days to get a decent history on.

Reply:
I just double checked this   moving to forward causes 
on markets I already have data on
error: getTrades() returned duplicate results
{ product_id: 'XRP-USDT', to: 1516785694047 }
{ product_id: 'XRP-USDT', to: 1516785694047 }
it returns nothing on markets I don't have data on.

So for now, I think it should remain as backward until I can looking into the history calls in bittrex.